LICENSE/LICENSE

```diff
@@ -0,0 +1,16 @@
+Copyright <2019>, The Apereo Foundation
```

Comment:
Could we please use the header as per what's in oEQ now. i.e. There should be no fixed dates in the headers, but instead a reference to NOTICE.

Also, does this have to be LICENSE/LICENSE rather than just LICENSE?